subprojects/publish/src/main/java/org/gradle/api/publish/internal/metadata/ModuleMetadataSpecBuilder.java

```diff
@@ -67,35 +61,36 @@
 import static java.util.Collections.emptyList;
 import static org.apache.commons.lang.StringUtils.isNotEmpty;
 
-class ModuleMetadataSpecBuilder {
+public class ModuleMetadataSpecBuilder {
```

Comment:
💭 Could you add a comment relating this to the `IvyComponentParser` and `MavenComponentParser`?  Otherwise it's not obvious they are performing a similar function.

Reply:
Add some javadoc to `ModuleMetadataSpecBuilder` and `ModuleMetadataSpec`